docs/guides/key-concepts.md

```diff
@@ -12,6 +12,26 @@ It is also common to apply quantizers to the weights during training. This is ne
 
 The true gradient of a quantizer is in general zero almost everywhere and therefore cannot be used for gradient descent. Instead, optimization of BNNs rely on what we call pseudo-gradients, which are used during back-propagation. In the documentation for each quantizer you will find the definition and a graph of the pseudo-gradient.
 
+### Using Quantizers as Activations
```

Comment:
Do you think it would be good to add a similar section [here](https://github.com/larq/larq/blob/master/larq/quantizers.py#L1) and [here](https://github.com/larq/larq/blob/master/larq/layers.py#L1) too? Very much like we do it for the [activations](https://github.com/larq/larq/blob/master/larq/activations.py#L1)

Reply:
I added a section to quantizers.py, but what exactly would you like to see in layers.py?

Reply:
Yeah, you are right. I though it would make sense to maybe explain the `input_quantizer` in the layers file too, since it is a layer kwarg.